src/flamenco/runtime/fd_account.h

```diff
@@ -159,6 +104,20 @@ fd_account_get_lamports( fd_account_meta_t const * meta ) {
   return meta->info.lamports;
 }
 
+static inline ulong
+fd_account_get_lamports2( fd_exec_instr_ctx_t const * ctx,
```

Comment:
This API is a bit ugly, suggest renaming it to `fd_account_get_lamports`

Reply:
The other API should probably be `fd_account_meta_lamports`

Reply:
I'll address in follow up PR